nodeServerless/Local CE SAM/CE_lambda_functions/card_get.js

```diff
@@ -0,0 +1,62 @@
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Context doc: https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html 
+ * @param {Object} context
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ * 
+ */
+
+let response;
+var AWS = require('aws-sdk');
+AWS.config.update({
+    region: "us-west-2"
+})
+
+let docClient = new AWS.DynamoDB.DocumentClient();
+// let ddb = new AWS.DynamoDB();
+
+exports.lambdaHandler = async (event, context) => {
+    try {
```

Comment:
Probably a bad idea to stuff everything in a big try/catch. Use try/catch intentionally and with precision.